README.md

```diff
@@ -10,7 +10,10 @@ annotations.
 - Enable workflows in the GitHub repository
 - Under [secrets](https://github.com/Sage-Bionetworks/amp-ad-metadata-dictionary/settings/secrets/actions) click 'New repository secret'
 - Enter secrets for `RSCONNECT_USER`, `RSCONNECT_TOKEN`, and `RSCONNECT_SECRET`, the values for which are saved in Sage's LastPass.
-- Trigger the GitHub action.
+- Push to new branch on Github
+- Create pull request with at least one reviewer to master branch
```

Comment:
I'd suggest updating this to be more explicit:
Create a pull request to merge your feature branc to master; at least one reviewer must approve this PR before it can be merged, which will trigger the staging deployment